### DISCUSSION TITLE
更新版本（2.4.4 快照版）后清理缓存导致所有游戏版本崩溃

### DISCUSSION THREAD

**Discussion by IceYi233** [Meloong-Git/PCL#644] [提问] [answered]
更新版本（2.4.4 快照版）后在“设置-启动器-系统”中点击“清理缓存”按键（图一），反应了好长一段时间后显示清理出14.3G的缓存。再次重启PCL2时发现 原文件夹.minecraft 中所有游戏版本均显示为“错误的版本”，在“版本选择”一栏中均显示为“未找到版本json文件”（图二）。

打开各版本（使用了版本隔离）文件夹后发现缺少一些文件（图三），且mods(模组), config(配置), resourcepacks(资源包), shader packs(光影，仅对有Optifine的版本), **screenshots(截屏)文件夹里的原始内容全部消失（图五）；saves(存档)里面各世界文件有所差异（图四）。**

在原文件夹.minecraft 中出现了一个名为“PCL”的配置设置（图六），打开后如图七，记录了之前所有版本的名称。（其实上面那个json文件也不记得是清完缓存后出现的还是老早以前装启动器时就有的；包括这个名为“PCL”的配置设置也不太清楚是不是装启动器时就已经有了，求大佬顺便帮忙看一下）

现在想知道是否是启动器版本更新导致了这个问题，如果是的话该怎样修复那些缺失的组件和游戏原文件？望各位大佬帮忙查看解决

![e99ca58466d99b76e119d8ee0ae01cc](https://user-images.githubusercontent.com/121037817/208455599-66141998-36d3-4d45-a6cd-ef7a355bedb6.jpg) 图一

![d3ab1e7e1719f1662af35e450db66d6](https://user-images.githubusercontent.com/121037817/208455688-bc75c1dd-fc44-4538-a51e-44b3ac8710d2.jpg) 图二

![cef6892c8c51b16cd1f71cb12e4791c](https://user-images.githubusercontent.com/121037817/208455811-430e1586-ea1b-4500-a521-0f0e14fcb8ab.jpg) 图三：整体对比；左为后来重新下的正常版本，右为清完缓存后有问题的版本

![52f7e2b9e5c9dbcd76d133ee82c92e5](https://user-images.githubusercontent.com/121037817/208456326-a0f9dff4-23c0-439e-9fee-7bd2d228956d.jpg) 图四：saves(存档)对比；左为有问题的版本，右为正常版本

![65111288aa60ef98d1af71fd403db07](https://user-images.githubusercontent.com/121037817/208456726-1fb8696a-52cf-4474-854a-e7455688b260.jpg) 图五：有问题的版本的mods文件夹

![c1277b1c4ee7536c7410efd569a17da](https://user-images.githubusercontent.com/121037817/208458425-5c724636-df83-42e5-aa83-c86cf69fdcbf.jpg) 图六：多出现的“PCL”配置设置

![3ec6fb73386e655ebd601beb205696e](https://user-images.githubusercontent.com/121037817/208458623-e553ca5b-8b3c-4e78-9303-77de1566bd59.jpg) 图七：“PCL”配置设置用记事本打开后

**Comment by Pigeon0v0** [comment]
手动重新安装.jpg

**Comment by Pigeon0v0** [comment, marked as answer]
#645
